cmd/shp/main.go

```diff
@@ -25,3 +27,24 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// goflags ignores -h and --help because pflag will show the usage later with all subcommands.
+func init_goflags() {
```

Comment:
Go has a handy feature called `godoc` where comments in front of function names automatically generate documentation. For this to work, the first word of the comment and the function name should match. Many linters catch this (and in the future we will probably add these lint checks to our CI).

```suggestion
// initGoFlags initializes the flag sets for klog. Any flags for "-h" or "--help" are ignored because pflag will show the usage later with all subcommands.
func initGoFlags() {
```